grid/chat/members/maps/discord.suma

```diff
@@ -0,0 +1,191 @@
+profile = "chat/members@1.0"
+provider = "discord"
+
+"""
+GetMembers map
+"""
+map GetMembers {
+  return map error if (input.workspace === undefined) {
+    title = "Input parameter `workspace` is required for provider discord"
+  }
+
+  http GET "/guilds/{input.workspace}/members" {
+    request "application/json" {
+      query {
+        after = input.page
+        limit = input.limit
+      }
+
+      headers {
+        "Authorization" = `Bot ${parameters.accessToken}`
+      }
+    }
+
+    response 200 "application/json" {
+      members = call MapMembers(members = body)
+      rateLimit = call MapRateLimitDetails(headers = headers)
+
+      return map result if ((members.length < input.limit) || (members.length === 0)) {
```

Comment:
This condition is technically correct, but confusing if `input.limit` is not set. Consider setting a default fallback value, perhaps something higher than Discord's default `1` (e.g. `100`).

Reply:
True, I'll have to fix it in other maps as well.